core/genesis.go

```diff
@@ -590,6 +590,18 @@ func DefaultIliadGenesisBlock() *Genesis {
 	}
 }
 
+// DefaultOdysseyGenesisBlock returns the odyssey network genesis block.
+func DefaultOdysseyGenesisBlock() *Genesis {
+	return &Genesis{
+		Config:     params.OdysseyChainConfig,
+		Difficulty: big.NewInt(0x20000),
+		GasLimit:   0x7A1200,
+		Nonce:      0x42,
+		Timestamp:  0,
+		Alloc:      decodePrealloc(odysseyAllocData),
```

Comment:
qq: are we gonna use the same alloc data with Iliad for Odyssey?

Reply:
The alloc data has not been modified on Odyssey.